app/views/courses/ClassroomSettingsModal.coffee

```diff
@@ -120,4 +126,22 @@ module.exports = class ClassroomSettingsModal extends ModalView
     @render()
     $('.google-class-name').hide()
     $('.class-name').show()
-    $('#link-google-classroom-btn').show()
\ No newline at end of file
+    $('#link-google-classroom-btn').show()
+
+  onPickImage: ->
+    filepicker.pick @onFileChosen
+
+  onFileChosen: (InkBlob) =>
```

Comment:
Is `InkBlob` an object or a class? We generally have the convention to use uppercase only for classes / schemas. That way it's clear and easy to read that a variable is actually a class / schema, and not an object we use.

Reply:
oh... i think that should be a blob object? (i copy paste this code too...) let me rename it to `inkBlob`